.gitignore

```diff
@@ -14,3 +14,4 @@ package-lock.json
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 xcuserdata/
 yarn-error.log*
+.vs/
```

Comment:
Do we still need this line if it's already specified in `windows/.gitignore`?

Reply:
Seems like it's needed. Will delete from windows/.gitignore and keen only here.